docs/griptape-tools/official-tools/inpainting-image-generation-client.md

```diff
@@ -1,6 +1,6 @@
 # InpaintingImageGenerationClient
 
-This tool allows LLMs to generate images using inpainting, where an input image is altered within the area specified by a mask image according to a prompt.
+This tool allows LLMs to generate images using inpainting, where an input image is altered within the area specified by a mask image according to a prompt. The input and mask images can be provided either by their filepaths or by their [Task Memory](../../griptape-framework/tools/task-memory.md) references.
```

Comment:
`filepaths` -> `file-paths` or `file paths`

Reply:
Updated to `file path` throughout